code/src/components/StartScreen.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import styled from 'styled-components';
+
+import games, { generateGame } from '../reducers/games';
+
+const StartButton = styled.button`
+  background-color: green;
+  border: none;
+  color: lightgreen;
+  border-radius: 50px;
+  font-size: 20px;
+  margin-top: 20px;
+  padding: 10px;
+`
+const InputField = styled.input`
+  border-color: lightgreen;
+  &:focus {
+    outline: 2px solid green;
+  }
+`
+
+const StartScreen = () => {
+    const [inputValue, setInputValue] = useState(''); 
+
+    const dispatch = useDispatch();
+
+    const onUserSet = () => {
+        dispatch(games.actions.setUsername(inputValue));
+        dispatch(generateGame());
+    }
+
+  return (
+      <div>
+          <h1>The Labyrith game</h1>
+          <div style={{marginBottom: '10px'}}>
+          <InputField
```

Comment:
For accessibility reason, I'd add a <label> tag to the form input or title="" in the InputField